contrib/kafka/src/test/java/com/google/cloud/dataflow/contrib/kafka/coders/ConsumerRecordCoderTest.java

```diff
@@ -0,0 +1,65 @@
+package com.workiva.cloud.dataflow.coders;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.dataflow.sdk.coders.Coder;
+import com.google.cloud.dataflow.sdk.coders.CoderException;
+import com.google.cloud.dataflow.sdk.util.Serializer;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Test case for {@link ConsumerRecordCoder}.
+ */
+@RunWith(JUnit4.class)
+public class ConsumerRecordCoderTest {
+
+  @Test
+  public void testDecodeEncodeEqual() throws Exception {
```

Comment:
I had meant to leave a comment or PR about this. The utilities in the repo only work if your element overrides equals/hashcode. I also cannot access the standalone encode/decode utilities because they are private to the package.